README.md

```diff
@@ -279,15 +279,16 @@ The key code for this purpose is:
 
 ## Certificate File
 
-Login to the [iOS Provisioning Portal (App IDs)](https://developer.apple.com/ios/manage/bundles/index.action).
+Login to the [iOS Provisioning Portal (App IDs)](https://developer.apple.com/account/ios/identifier/bundle).
 
-Configure the appropriate certificate for push notifications and download the
-certificate:
+Select your app bundle, scroll down and click "Edit." This will bring you to a page that will allow you to setup and configure services for your app. Scroll down until you see the service labeled “Push Notifications.” Depending on whether you’re creating a Development or Production certificate, click the appropriate “Create Certificate” button.
 
-![Downloading the Push Notification
-Certificate](images/Downloading_the_Push_Notification_Certificate.png)
+![Creating the Push Notification
+Certificate](https://cdn-images-1.medium.com/max/1600/1*zHi-WR44OGYtOdFCy79jZw.png)
```

Comment:
Can we add this image to this repo? Or capture one that we can? Hot-linked images tend to break as time goes on.